mne/preprocessing/stim.py

```diff
@@ -67,9 +78,16 @@ def fix_stim_artifact(inst, events=None, event_id=None, tmin=0.,
     inst : instance of Raw or Evoked or Epochs
         Instance with modified data
     """
-    _check_option('mode', mode, ['linear', 'window'])
+    _check_option('mode', mode, ['linear', 'window', 'constant'])
     s_start = int(np.ceil(inst.info['sfreq'] * tmin))
     s_end = int(np.ceil(inst.info['sfreq'] * tmax))
+    if mode == "constant" and baseline is None:
+        raise ValueError('Please provide the baseline for mode="constant"')
```

Comment:
use the function `[_check_baseline](https://github.com/mne-tools/mne-python/blob/e18950e9c8635997139c3cc4d940512a64dc5723/mne/epochs.py#L1701)` to make sure that `base_t1` and `base_t2` provided by the user are good

Reply:
It's a bit more complicated than this because this function can also operate on `raw` instances.

Given this I'm just going to make it take a tuple of length 2 containing floats only, a follow-up PR could make it support `None` for Epochs if it makes sense. But in the meantime I don't mind the extra explicitness!